frontend/src/component/feature/CopyFeature/CopyFeature.tsx

```diff
@@ -7,18 +7,24 @@ import {
     Paper,
     FormControlLabel,
     Alert,
+    styled,
 } from '@mui/material';
 import { FileCopy } from '@mui/icons-material';
 import { styles as themeStyles } from 'component/common';
 import { formatUnknownError } from 'utils/formatUnknownError';
-import styles from './CopyFeature.module.scss';
+// import styles from './CopyFeature.module.scss';
```

Comment:
Needs converting to styled components. Maybe I'll try to have a go at it on this PR before we merge it.

Reply:
Addressed in https://github.com/Unleash/unleash/pull/3298/commits/bf8b677668c3d0cf2cdf20295e31792c8fda8b50.